docs/quick-start.md

```diff
@@ -0,0 +1,192 @@
+CERTCHAIN_PATH=${KEY_NAME}-chain.crt
+cat ${SIGNED_CERT_PATH} ca.crt > ${CERTCHAIN_PATH}
+```
+
+## Upload the leaf certificate to Azure KeyVault
+```bash
+az keyvault certificate pending merge --vault-name ${AKV_NAME} --name ${KEY_NAME} --file ${CERTCHAIN_PATH}
+```
+
+## Sign the container image:
+
+Log in to the ACR:
+
+```bash
+export NOTATION_PASSWORD=$(az acr login --name ${REGISTRY} --expose-token --output tsv --query accessToken)
+```
+
+Get the Key ID for the certificate and add the Key ID to the keys and certs:
+
+```bash
+KEY_ID=$(az keyvault certificate show -n ${KEY_NAME} --vault-name ${AKV_NAME} --query 'kid' -o tsv)
+notation key add --name ${KEY_NAME} --plugin azure-kv --id ${KEY_ID}
+notation key ls
+```
+
+Choose an authorization mode to visit your keyvault from the plugin:
+
+- Option 1: Authorized by Managed Identity (by default):
+    ```bash
+    export AKV_AUTH_METHOD="AKV_AUTH_FROM_MI"
+    ```
+
+    Make sure you have at least granted secret-get, certificate-get, key-sign permissions to your resources.
+
+    The script below is a demo to check and set your Azure VM's access policy to your keyvault. For more details, please check [this guide](https://docs.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal) for details.
+    
+    ```bash
+    # get your azure vm's principalId
+    az vm list --query "[?name=='${VM_NAME}'].identity"
+
+    # get your keyvault's access policy for your azure vm. $OID should be the principalId which we get from the above command
+    az keyvault show --name ${AKV_NAME} --query "properties.accessPolicies[].{objectId:objectId,permissions:permissions}[?contains(objectId,'${OID}')]"
+
+    # if policy doesn't meet the sign requirements, set the access policy.
+    az keyvault set-policy --name ${AKV_NAME} --object-id ${OID} --secret-permissions get --key-permissions sign --certificate-permissions get
+    ```
+    
+- Option 2: Authorized by Azure CLI 2.0:
+
+    ```bash
+    export AKV_AUTH_METHOD="AKV_AUTH_FROM_CLI"
+    ```
+    Login to your azure account by Azure CLI
+    ```bash
+    az login
+    az account set --subscription ${subscriptionID}
+    ```
+
+Sign the container image:
+
+```bash
+notation sign --key ${KEY_NAME} ${IMAGE}
+```
+
+List the signatures:
+
+```bash
+notation ls ${IMAGE}
+```
+
+## Verify the container image
+
+Download the certificate:
+
+```bash
+CERT_ID=$(az keyvault certificate show -n ${KEY_NAME} --vault-name ${AKV_NAME} --query 'sid' -o tsv)
+CERT_PATH=${KEY_NAME}-cert.crt
+az keyvault secret download --file ${CERT_PATH} --id ${CERT_ID}
+```
+
+Generate a CA certificate:
+
+```bash
+notation cert add --name ${KEY_NAME} ${CERT_PATH}
+notation cert ls
+```
+
+Verify the container image:
+
+```bash
+notation verify --cert ${KEY_NAME} ${IMAGE}
+```
+
+You can use the notation verify command to ensure the container image hasn't been tampered with since build time by comparing the `sha` with what is in the registry.
```

Comment:
This sentence confuses me.

> You can use the notation verify command to ensure the container image hasn't been tampered with since build time 

makes sense to me but

> by comparing the `sha` with what is in the registry.

The above part confuses since the `sha` is always consistent with what is in the registry even if it has been tampered.

Reply:
Fixed.